src/nvim/charset.c

```diff
@@ -734,7 +734,7 @@ int vim_strnsize(char_u *s, int len)
 /// @return Number of characters.
 #define RET_WIN_BUF_CHARTABSIZE(wp, buf, p, col) \
   if (*(p) == TAB && (!(wp)->w_p_list || lcs_tab1)) { \
-    int ts; \
+    int64_t ts; \
```

Comment:
I thought I wasn't supposed to use `long` type as written in the [style guide](https://neovim.io/develop/style-guide.xml?showone=Integer_Types#Integer_Types).
I added the cast `ts = (int)(buf)->b_p_ts` :)